vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -140,6 +145,25 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
         }
     }
 
+    private fun updateStoppedBeaconInfo(roomId: String) {
```

Comment:
To clarify namings what do you think about renaming this method to `sendStoppedBeaconInfo()` and the method `sendBeaconInfo()` to `sendLivedBeaconInfo()` ?

Reply:
Done.